### ISSUE TITLE
Get parameters in "I go to ..." won't work.

### ISSUE THREAD

**Issue by cabello** [heynemann/pyccuracy#42]
When trying to go to a page with get parameters like this scenario above we receive an error.

```
Scenario 1 - Viewing an article
   Given
       I go to "http://localhost/project/page.py?article=1&lang=us"
   When
       I wait for 1 seconds
   Then
       I see "link_exit" div contains "Exit"
```

Here is the error that I am receiving:

```
Error Description
-----------------
The action line does not match any of the registered actions.

If you just need a text like "I wait for the page to finish loading"
or something like this, prefix your line with a # sign.
```

When I remove the get parameters the pyccuracy runs.

**Comment by cabello** [comment]
I think this problem happens because the regular expression in page_go_to_regex variable is wrong in both languages/data/(en-us.txt|pt-br.txt), I don't know about the selenium support of get parameters, but fixing the regex might fix the problem.

I found a page with some regular expression for urls[1], I know you won't use the regex on the page, but just to remember that some urls have ~, # and another especial characters.

[1] - http://www.geekzilla.co.uk/view2D3B0109-C1B2-4B4E-BFFD-E8088CBC85FD.htm

**Comment by heynemann** [comment]
Fixed in release 1.2.24
